libraries/AC_Avoidance/AP_OABendyRuler.cpp

```diff
@@ -97,9 +107,29 @@ bool AP_OABendyRuler::update(const Location& current_loc, const Location& destin
     } else {
         ground_course_deg = degrees(ground_speed_vec.angle());
     }
+    
+    bool state;
+    switch (get_type()) {
+        case OABendyType::OA_BENDY_VERTICAL:
+        #if APM_BUILD_TYPE(APM_BUILD_ArduCopter)
```

Comment:
As mentioned yesterday, could we make a definition at the top of this .cpp file called something like below and then use it in place of, "#if APM_BUILD_TYPE(APM_BUILD_ArduCopter)".  It's a non-functional change and I think this will make it more obvious to future developers that this "switch" exists.  Without it I can imagine that plane or sub developers may lose time trying to figure out why they can't get vertical bendyruler working.

#define VERTICAL_ENABLED  APM_BUILD_TYPE(APM_BUILD_ArduCopter)